scripts/autoregress.py

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/env python
+
+# This is a script to test ASF MapReady against reference data that is known to
+# be correct. It is run on a directory containing subdirectories which contain
+# tests on which MapReady will be run. The results of MapReady are then compared
+# against reference data containing '.ref' just before the extension.
+
+import os
+import argparse
```

Comment:
This module isn't available in Python 2.6, which is all we have on the RHEL/CentOS 6 systems ASF uses.